test/active_model_serializers/railtie_test_isolated.rb

```diff
@@ -18,6 +18,11 @@ class WithRails < RailtieTest
         "ActionController::Serialization should be included in ActionController::Base, but isn't"
     end
 
+    test 'mixes ActiveModelSerializers::Support::UrlHelpers::Rails into ActiveModelSerializers::Support::UrlHelpers' do
+      assert ActiveModelSerializers::Support::UrlHelpers.constants.include?(:Rails),
+        'ActiveModelSerializers::Support::UrlHelpers::Rails should be defined but is not'
+    end
```

Comment:
- [ ] would be good to test behavior of `url_for` and `default_url_options` here and below